src/libs/actions/Policy/Policy.ts

```diff
@@ -1947,9 +1956,24 @@ function buildPolicyData(policyOwnerEmail = '', makeMeAdmin = false, policyName
         expenseCreatedReportActionID,
         customUnitID,
         customUnitRateID,
-        engagementChoice,
     };
 
+    if (!introSelected?.createWorkspace && engagementChoice) {
+        const {
+            guidedSetupData,
```

Comment:
For some reason when I test locally, `guidedSetupData` is always `[]` here when I manually create a workspace after choosing the onboarding intent "Track and budget expenses" but I can't figure out why 🤔